### ISSUE TITLE
Global accessibility: lang attribute is missing

### ISSUE THREAD

**Issue by swickr** [w3c/validator-suite#185]
<code>lang</code> attribute is missing

There is no language defined for this document 

From [Mark's full assessment](https://lists.w3.org/Archives/Team/team-vs/2013Aug/att-0010/ValidatorSuiteA11y.html)
